src/ESLint.Formatter/sarif.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * @fileoverview SARIF v2.0 formatter
+ * @author Chris Meyer
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Helper Functions
+//------------------------------------------------------------------------------
+
+/**
+ * Returns the severity of warning or error
+ * @param {Object} message message object to examine
+ * @returns {string} severity level
+ * @private
+ */
+function getResultLevel(message) {
+    if (message.fatal || message.severity === 2) {
```

Comment:
>message.severity === 2) [](start = 25, length = 23)

severity 2 is error? what is severity == 1?

Reply:
1 == warning

---
In reply to: [249136739](https://github.com/Microsoft/sarif-sdk/pull/1222#discussion_r249136739) [](ancestors = 249136739)